app/subscriber/src/features/my-minister/MyMinister.tsx

```diff
@@ -25,10 +26,12 @@ export const MyMinister: React.FC = () => {
           makeFilter({
             ...filter,
             startDate: '',
+            contentTypes: [],
             endDate: '',
           }),
         );
-        setHomeItems(data.items);
+        // don't want to keyword fetch when there is nothing to fetch
+        setHomeItems(userInfo?.preferences?.myMinisters.length > 0 ? data.items : []);
```

Comment:
This was causing a bug that was returning all content when the user had no ministers selected

Reply:
(since it is tied to keyword and the keyword would be '')